index.d.ts

```diff
@@ -1,10 +1,11 @@
+export interface NormalizedWheelEvent {
```

Comment:
exporting typing will make `declare module` fail

Reply:
sorry, I didn't take that into consideration.

Reply:
I fixed it in the new commit.

Reply:
Thanks, This also works:

```ts
callback: (e: WheelEvent, normalized: ReturnType<typeof normalizeWheel>) => void
```